addons/mail/static/src/models/thread_view_topbar/thread_view_topbar.js

```diff
@@ -465,6 +464,31 @@ function factory(dependencies) {
             );
         }
 
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeHasDescription() {
+            return Boolean(
+                this.thread &&
+                (this.thread.description ||
+                this.thread.isChannelDescriptionChangeable)
+            );
+        }
+
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeIsDescriptionEditable() {
+            return Boolean(
+                this.isMouseOverThreadDescription &&
```

Comment:
This field name does not match the compute code.

The fact of being editable should be unrelated to the position of the mouse.